OTOnexus/Classes/Capture/OTOCaptureView.swift

```diff
@@ -555,7 +589,8 @@ extension OTOCaptureView {
     fileprivate func processBarcode(_ metadataObjects: [AnyObject]) -> String {
         if let barcodeObj = metadataObjects[0] as? AVMetadataMachineReadableCodeObject,
             supportedBarCodes.contains(barcodeObj.type) && barcodeObj.stringValue != nil {
-            drawBarcodeBounds(barcodeObj)
+            scannedCodeObject = barcodeObj
+            drawBarcodeBounds()
```

Comment:
Did you consider passing `barcodeObj` into `drawBarcodeBounds()` and friends? It would avoid the guard checks at the beginnings of those functions. I didn't see anywhere else the property is being used.

Reply:
well, I did consider it, but for the `barcodeOrientation` I didn't really have the barcode handy so I just decided to capture it and make it consistent.